sources/packages/backend/libs/services/src/students-assessments/assessment-sequential-processing.service.models.ts

```diff
@@ -124,3 +124,28 @@ export interface AwardTotal {
   valueCode: string;
   total: number;
 }
+
+/**
+ * Program year totals with award code (e.g. CSPT, CSGD, CSGP, SBSD, BCAG) and its totals promise
```

Comment:
Please review the comments, for instance, the properties are no longer promises.